src/azure-cli/setup.py

```diff
@@ -87,7 +87,7 @@
     'azure-mgmt-devtestlabs~=4.0',
     'azure-mgmt-dns~=8.0.0',
     'azure-mgmt-eventgrid==10.2.0b2',
-    'azure-mgmt-eventhub~=10.1.0',
+    'azure-mgmt-eventhub~=11.0.0b1',
```

Comment:
Please fix the bate version in setup.py

Reply:
Hey @kairu-ms , I am using the latest python sdk release version here. i.e 11.0.0b1.
https://github.com/Azure/azure-sdk-for-python/pull/29777